local-cli/rnpm/windows/package.json

```diff
@@ -21,7 +18,7 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "react-native": ">=0.31.0"
+    "react-native": "~0.49.0"
   },
```

Comment:
Why does this require a specific react-native version? Shouldn't this plugin be compatible across many versions of react-native? I don't want to have to point people to specific versions of rnpm-plugin-windows depending on which version of react-native they are using.

Reply:
Would it be okay to change this to >= 0.40 (or another version)? I'd like to update this to be closer to where we are in the react-native-windows project.